features/cellular/framework/targets/GEMALTO/CINTERION/GEMALTO_CINTERION.cpp

```diff
@@ -147,6 +149,23 @@ void GEMALTO_CINTERION::init_module_ems31()
     _module = ModuleEMS31;
 }
 
+void GEMALTO_CINTERION::init_module_ehs5e()
+{
+    // EHS5-E
+    static const intptr_t cellular_properties[AT_CellularBase::PROPERTY_MAX] = {
+        AT_CellularNetwork::RegistrationModeDisable,    // C_GREG
```

Comment:
This should probably be like below due to EHS5-E seems to be:
- 2G/3G modem
- With support for IPv4 and IPv6 but not for dual-stack
```
        AT_CellularNetwork::RegistrationModeDisable,    // C_EREG
        AT_CellularNetwork::RegistrationModeLAC,    // C_GREG
        AT_CellularNetwork::RegistrationModeLAC,    // C_REG
...
        1,  // PROPERTY_IPV4_STACK
        1,  // PROPERTY_IPV6_STACK
        0,  // PROPERTY_IPV4V6_STACK
```

Reply:
You are right the registration modes need to be fixed. But i think the module does support dual stack mode

Reply:
About dual-stack, according to `ehs5-e_atc_v03001.pdf` connection must be GPRS0 or GPRS6 and the current driver creates only one internet connection so it can't handle dual-stack yet.

Reply:
Ok, im going to disable it.

Reply:
@ghseb not for `ModuleEMS31` :)